File: lecture_002(Normal)/l_014_benjaminbulbs.cpp
#include<iostream>
using namespace std;

void benjaminbulbs(int n) {   //n=no. of bulbs or no. of fluctuations
    cout<<"Bulbs that would be switched on are: ";
    int count = 0;
    for(int i=1; i*i<=n; i++) {
        cout<<i*i<<endl;
        count++;
    }
    cout<<"No of bulbs that would be switched on are: "<<count<<endl;
}
int main(int args, char**argv) {
    int n;
    cin>>n;
    benjaminbulbs(n);
    
    return 0;
}

File: lecture_002(Normal)/l_014_benjaminbulbs.java
import java.util.Scanner;
public class l_014_benjaminbulbs {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        int n = scn.nextInt();
        benjaminbulbs(n);
    }
    public static void benjaminbulbs(int n) {   //n=no. of bulbs or no. of fluctuations
        System.out.print("Bulbs that would be switched on are: ");
        int count = 0;
        for(int i=1; i*i<=n; i++) {
            System.out.println(i*i);
            count++;
        }
        System.out.println("No of bulbs that would be switched on are: "+ count);
    }
}